brave-asynchttpclient/brave-asynchttpclient-2.x/src/main/java/smartthings/brave/asynchttpclient/ClientTracing.java

```diff
@@ -32,7 +32,7 @@
 import org.asynchttpclient.filter.IOExceptionFilter;
 import org.asynchttpclient.filter.RequestFilter;
 import org.asynchttpclient.filter.ResponseFilter;
-import zipkin.Endpoint;
+import zipkin2.Endpoint;
```

Comment:
ps this could break your call sites, so if a problem lemme know and I can help make it defensive (ex convert old endpoint to new one). Otherwise on call-site do `endpoint.toV2()`

Reply:
Thanks for the heads up.  I don't think it will be a problem but will do some code review before merging this.